pkg/api/types.go

```diff
@@ -2213,6 +2213,17 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "SYSTEM" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by
+	// creating a PriorityClass object with that name.
+	// If not specified, the pod priority will will be default or zero if there is
```

Comment:
`will be default or zero if there is no default`.

Are we going to support `default` now? I found that in the design document, it is `will`, just want to confirm if the `default` will be available in the first implementation? If not, how about add some words such as `will` to identify it is future work?